src/components/IOUConfirmationList.js

```diff
@@ -101,39 +102,92 @@ const defaultProps = {
 const MINIMUM_BOTTOM_OFFSET = 240;
 
 class IOUConfirmationList extends Component {
+    constructor(props) {
+        super(props);
+
+        this.toggleOption = this.toggleOption.bind(this);
+
+        const participants = this.props.participants.map(participant => ({
+            ...participant, selected: true,
+        }));
+
+        const formattedParticipants = this.getParticipantsWithAmount(participants);
```

Comment:
I think you could save a little bit of space here by doing the following, since we only use the `participants` variable on line 110 once, right?

```js
const formattedParticipants = this.getParticipantsWithAmount(this.props.participants).map(participant => ({
    ...participant, selected: true,
}));
```

Reply:
Yeah